src/main/java/com/notably/logic/parser/OpenCommandParser.java

```diff
@@ -55,6 +60,7 @@ public OpenCommandParser(Model notablyModel, CorrectionEngine<AbsolutePath> path
             throw new ParseException(String.format(ERROR_PATH, titles));
         }
 
+        logger.info("OpenCommand created");
```

Comment:
Perhaps you can be more explicit here. There are a lot of paths of execution in this `OpenCommandParser#parse` execution. It'll be better if you can fine-tune the logger to log out the exact detail given the different paths that are taken!